### PR TITLE
Improve Github Issue Templates

### PR DESCRIPTION
**Pull request by travelton**
- Improved Github Issue templates to ensure we are capturing information consistently between issue types.
- Fixed formatting of labels, to ensure labels are automatically applied.
- Updated a few links to new domain.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -10,22 +10,43 @@ labels: "type: bug"
...
+- Tsh version (`tsh version`):
+- Computer OS (e.g. Linux, macOS, Windows):
+- Browser version (for UI-related issues):
+- Installed via (e.g. apt, yum, brew, website download):
+- Additional details:
+
+### Debug Logs
+Please include or attach debug logs, when appropriate. Obfuscate sensitive information!
+- Start Teleport with --debug flag (`teleport --debug`)
+- Run Tsh with --debug flag (`tsh --debug`)
```
Comment (webvictim) [inline .github/ISSUE_TEMPLATE/bug_report.md:36]:
```suggestion
- Run tsh with --debug flag (`tsh --debug`)
```

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 2]
```diff
@@ -1,14 +1,27 @@
 ---
 name: Documentation 📝
-about: Suggest better documentation for a particular tool or process. 
-labels: "type: documentation"
+about: Suggest improvements to Teleport's documentation site.
```
Comment (webvictim) [inline .github/ISSUE_TEMPLATE/documentation.md:3]:
```suggestion
about: Suggest improvements to Teleport's documentation.
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -1,26 +1,34 @@
...
-  - Documentation: https://gravitational.com/teleport/docs/
-
-  - If you're an OSS user and have a general question, please ask in our community forum: https://community.gravitational.com/c/teleport
+  - Documentation: https://goteleport.com/teleport/docs/
 
-  - For Pro and Enterprise users, please open a ticket via the Enterprise support portal: https://dashboard.gravitational.com/web/
-
-  Before opening a new issue, please search existing issues: https://github.com/gravitational/teleport/issues
+  - For Open Source users, please ask in our community forum: https://community.goteleport.com/c/teleport
+  - For Enterprise customers, please open a support ticket via the Enterprise support portal: https://dashboard.gravitational.com/web/
```
Comment (webvictim) [inline .github/ISSUE_TEMPLATE/question.md:13]: Given that we're giving these two options, is there any point in having this issue type at all?

**Review by webvictim — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -10,22 +10,43 @@ labels: "type: bug"
...
+
+
+### Which platform(s)
+- [ ] Teleport Open Source
+- [ ] Teleport Cloud
+- [x] Teleport Enterprise
+- [ ] Other
+
+### Which component(s)
```
Comment (benarent) [inline .github/ISSUE_TEMPLATE/bug_report.md:45]: This is handy.

**Review by benarent — APPROVED** [review]
Looks good, I just merged other updates earlier today that cleaned up some Gravitational vs GoTeleport issues.

File: .github/ISSUE_TEMPLATE/question.md (hunk 3 above)
Comment (travelton) [reply .github/ISSUE_TEMPLATE/question.md:13]: In `config.yml`, perhaps we add a support option? Then eliminate `question.md`. I've only seen it used once. 

- Teleport Documentation
- Teleport Community Forum
- **Teleport Enterprise Support** -> https://dashboard.gravitational.com/web/
- Request a Demo

Comment (webvictim) [reply .github/ISSUE_TEMPLATE/question.md:13]: Great idea!

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -3,7 +3,7 @@ contact_links:
   - name: Teleport Documentation
     url: https://goteleport.com/teleport/docs/
     about: Check out the Teleport documentation for answers to common questions
-  - name: Teleport community forum
+  - name: Teleport Community Forum
     url: https://community.goteleport.com/
```
Comment (russjones) [inline .github/ISSUE_TEMPLATE/config.yml:7]: Update to GitHub discussions.

**Review by russjones — APPROVED** [review]
@travelton One comment, then please merge.

**Comment by benarent** [comment]
In hindsight the check list is making the issues look pretty strange? I wonder if we can keep something but not bubble it up to the Github list? 

image

**Comment by travelton** [comment]
@benarent That's annoying. I don't see any way around this with task lists. 

What if we just include a list, instead of a task list. Then, the user can delete items that are not applicable. 

```
### Which platform(s)
- Teleport Open Source
- Teleport Cloud
- Teleport Enterprise
- Other

### Which component(s)
- Server Access
- Kubernetes Access
- Application Access
- Database Access
- CLI Clients
- Web Clients
- Other
```

**Comment by benarent** [comment]
^ Yeah, I think that's a good idea. You can add a markdown comment that'll let people know

**Pull request merged**